easybuild/easyconfigs/m/makedepf90/makedepf90-2.8.8-foss-2015a.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'makedepf90'
+version = '2.8.8'
+
+homepage = 'http://www.iki.fi/erik.edelmann/makedepf90'
```

Comment:
url doesn't exist anymore, is there another one?

Reply:
Hi. That url is the one that appears in official sources, but since it is not available I think it will be better to leave it empty because I did not find another official page.

Reply:
Or ask Erik where it is now if it still exists.

Reply:
Following your suggestion I just wrote to Erik right now, thanks for the suggestion akesandgren.

Reply:
Hi. I need to know if any other change is necessary in this easyblock. Well, I'm waiting for it to be accepted in order to continue working in another (FVCOM)